config/initializers/pre_wings.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.config.to_prepare do
+  WINGS_CONCERNS = [AdminSet, Collection, ConferenceItem, Dataset, Etd, ExamPaper, GenericWork, Image, JournalArticle, Oer, PublishedWork, Thesis].freeze
```

Comment:
Unless OER and ETD are actually fixed to work, I'd leave these out of the list... Adventist doesn't use them and so far they haven't worked

Reply:
they dont work because they aren't listed here. they worked fine as soon as I added them.